Salary App/Features/SalaryForm/Model/Calculators/NICalculator.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  NICalculator.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+struct NICalaculator {
+    
+    func calculate(salary: Double) -> Double {
+       // £30,000 - £9,568 = £20,432
+      // (£20,432 * 0.12) / 12 = £204.32
+
+        let ni = salary - 9568
```

Comment:
What is `9568`? Try to avoid magic numbers and create properties or leave comments explaining what these values represent so it's clear